samples/generated_samples/retail_v2_generated_catalog_service_add_catalog_attribute_async.py

```diff
@@ -23,7 +23,6 @@
 #   python3 -m pip install google-cloud-retail
 
 
-# [START retail_v2_generated_CatalogService_AddCatalogAttribute_async]
```

Comment:
Should we just skip the generated samples?

Reply:
@kchertenko Please could you revert the changes to the autogenerated samples?

Reply:
done

Reply:
SGTM now. Thanks